cli/config/configuration-fc.toml.in

```diff
@@ -0,0 +1,291 @@
+# Copyright (c) 2019 Intel Corporation
```

Comment:
@grahamwhaley - I wonder if this should show "2017-2019" since it's based on a file that already existed (and was introduced in 2017)?

Reply:
It could, and that would also be fine. Your call - if you feel more comfortable then pls go ahead and make the change :-)

Reply:
Thanks - updated!